docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,59 @@
+# XML Features
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlash.gif)
+
+Using a `>` at the end of an opening tag generates the corresponding closing tag.
+
+![Auto Close Tag End](../images/Features/AutoCloseTagEnd.gif)
+
+## Linked Editing
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) supports linked editing to simultaneously change an open and closing tag pair.
+
+![Linked Editing](../images/Features/LinkedEditing.gif)
+
+## Jump to Start/End Tag
+
+For quick navigation, you can use `Ctrl + Click` on a closing tag to jump to the corresponding opening tag.
+
+![Jump To Opening Tag](../images/Features/JumpToOpeningTagXML.gif)
+
+The same applies to jumping from an opening tag to the respective closing tag.
+
+![Jump To Closing Tag](../images/Features/JumpToClosingTagXML.gif)
+
+## Rename Tag
+
+To rename a single tag and its corresponding opening/closing, highlight the tag and press `F2` or use the dropdown to select `Rename Symbol`.
+
+![Rename Tag](../images/Features/RenameTagXML.gif)
+
+## Completion based on XSD
+
+With an existing XSD file bond to an XML file, completion based on tags defined in the XML Schema/XSD file are supported.
+
+![Completion Based On XSD](../images/Features/CompletionBasedOnXSD.gif)
+
+## Generate XSD or DTD From XML
```

Comment:
Replace with 

## Generate a grammar From XML

Add 2 sections Generate a XSD from XML and generate a DTD from XML.